### PR TITLE
Add Shopware 6.6.6.1

### PR DESCRIPTION
**Pull request by M-arcus**

### CODE REVIEW

File: tests/svrunit/suites/dev/6.6.6.0.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <testsuite name="dockware/dev:6.6.6.0, Core Checks" group="core" dockerImage="dockware/dev:6.6.6.0">
             <directory>./../../tests/shared/base</directory>
             <directory>"./../../tests/packages/ubuntu/22"</directory>
-            <directory > ./../../tests/shared/base-6.5</directory>
```
Comment (boxblinkracer) [inline tests/svrunit/suites/dev/6.6.6.0.xml:11]: hi there
not quite sure about this? did you manually change it or was is generated?

**Comment by boxblinkracer** [comment]
thank you for this
will merge it, but curious about the base-6.0 change :)

File: tests/svrunit/suites/dev/6.6.6.0.xml (hunk 1 above)
Comment (M-arcus) [reply tests/svrunit/suites/dev/6.6.6.0.xml:11]: This was generated via the command `make generate-tests` inside the `build` directory.

Comment (M-arcus) [reply tests/svrunit/suites/dev/6.6.6.0.xml:11]: I have adjusted these manually now

Comment (boxblinkracer) [reply tests/svrunit/suites/dev/6.6.6.0.xml:11]: ok perfect, ill check it out next time, seems weird

thanks again for this

Comment (M-arcus) [reply tests/svrunit/suites/dev/6.6.6.0.xml:11]: @boxblinkracer 
I have added a PR for that in https://github.com/dockware/dockware/pull/233
It was a check that checked for 6.5 in the string, which worked for 6.6.5.x but not for 6.6.6.1

**Pull request merged**